### ISSUE TITLE
App Store Connect developer team

### ISSUE THREAD

**Issue by kiranvenkatesh** [expo/expo#2530]
Hi,
I am first time user of Expo and preparing an app for test flight

I am currently trying to run **expo build:ios**

My company has an App Store Connect developer team and I have been assigned the role of "App Manager", using my Apple developer id. However, I am unable to submit any builds. I have been told I do not require another developer license because of the developer team.

<img width="590" alt="screen shot 2018-10-27 at 11 49 58 pm" src="https://user-images.githubusercontent.com/5430123/47612995-5a73c280-da43-11e8-868b-61ea421063d7.png">

The above is what I am seeing. What steps am I missing? Thank you in advance.

**Comment by brentvatne** [comment]
cc @dsokal / @jakubste

**Comment by jakubste** [comment]
The message in the beginning suggest that you're still using `exp `. Please update to `expo-cli` and check if the issue is still there.
Also: can you test this command with an account with higher permissions to Apple team?

**Comment by jakubste** [comment]
https://forums.expo.io/t/apple-store-connect-exp-build-ios/15282
Isn't this same issue?

**Comment by kiranvenkatesh** [comment]
First off, I apologize for duplicating this. I posted the same in the expo forums. This is primarily an issue with IT not supplying me with the correct enterprise developer credentials. I am working on retrieving that. In the meantime, I purchased a separate license and have been able to submit builds and upload to test flight. I will close this as it isn't an issue rather a lack of understanding the developer vs enterprise developer licenses. 

Here is the active topic that I will update once I am able to do this using an enterprise account.

https://forums.expo.io/t/apple-store-connect-exp-build-ios/15282
